src/clr-angular/modal/modal.html

```diff
@@ -22,7 +22,7 @@
 
         <div class="modal-content">
           <div class="modal-header">
-            <button type="button" class="close" *ngIf="closable" (click)="close()">
+            <button type="button" [attr.aria-label]="commonStrings.close" class="close" *ngIf="closable" (click)="close()">
               <clr-icon shape="close" [attr.title]="commonStrings.close"></clr-icon>
```

Comment:
I don't think we need the title on the icon anymore.

Reply:
Done ;)